suites/all.go

```diff
@@ -2,8 +2,13 @@ package suites
 
 import (
 	"github.com/dedis/kyber/group/edwards25519"
+	"github.com/dedis/kyber/pairing/bn256"
 )
 
 func init() {
 	register(edwards25519.NewBlakeSHA256Ed25519())
+	register(bn256.NewSuite().G1().(Suite))
```

Comment:
These should not be registerd in all.go because they are not constant time.

(the question of whether we should unify both is something maybe we should ask ourselves somewhere else, and I'm thinking that it'd be best if we can unify both and have a way to extract information out of a group, for example if it is constant time or not).